server/db/schema.models.ts

```diff
@@ -24,7 +24,11 @@ const dailyEntriesSchema = new mongoose.Schema({
     index: true,
     required: true,
   },
-  foodItems: [foodItemsSchema],
+  foodItem_ids: {
+    type: String,
+    index: true,
+    default: "",
+  },
```

Comment:
Please discuss all schema changes with the team before making them. I remember that you said you wanted to keep the nested list?  Not sure why we need this. 

I think we do need a fooditems id, this is currently disabled in the schema in line 18. I will re-enable it in my next PR.